pkg/virt-config/virt-config.go

```diff
@@ -107,6 +107,10 @@ func IsS390X(arch string) bool {
 	return arch == "s390x"
 }
 
+func (c *ClusterConfig) IsDownwardMetricsEnabled() bool {
```

Comment:
`IsDownwardMetricsEnabled` and `DownwardMetricsEnabled` are very similar names that can easily be confused. Can you give it another name?